ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -15,21 +15,55 @@ if get_option('enable-nnfw-runtime')
 
   nnfw_plugin_lib = shared_library('nnstreamer_filter_nnfw',
     nnstreamer_filter_nnfw_sources,
+    cpp_args: ['-Wno-sign-compare'],
     dependencies: nnstreamer_filter_nnfw_deps,
     install: true,
     install_dir: filter_subplugin_install_dir
   )
   static_library('nnstreamer_filter_nnfw',
     nnstreamer_filter_nnfw_sources,
+    cpp_args: ['-Wno-sign-compare'],
```

Comment:
How about setting this argument as the default one in the root meson.build?

Reply:
Yes, added a new PR #1984 for this, as it was out of scope for this PR.

Reply:
These args have removed. PR #1984 removes from other extensions as well.